### PR TITLE
CurveIn Arrow

### PR DESCRIPTION
**Pull request by dmoroz0v**
Screenshot 2023-05-09 at 21 24 36

### CODE REVIEW

**Review by bring-shrubbery — DISMISSED** [review]
Nice! I don't mind the naming, I think when someone needs to add more shapes they can rename it. Thanks for the extra previews by the way.

File: Sources/SwiftUITooltip/TooltipModifier.swift [hunk 1]
```diff
@@ -133,20 +133,34 @@ struct TooltipModifier<TooltipContent: View>: ViewModifier {
...
+    private func arrowShape(angle: Double, borderColor: Color? = nil) -> some View {
+        switch config.arrowType {
+        case .default:
+            let shape = ArrowShape()
+                .rotation(Angle(radians: angle))
+            if let borderColor {
+                return shape.stroke(borderColor)
+            }
+            return shape
+        case .curve:
```
[the remark this answers is not in the document]
Comment (dmoroz0v) [reply Sources/SwiftUITooltip/TooltipModifier.swift]: Fixed)

**Comment by dmoroz0v** [comment]
@bring-shrubbery hi. pr is ready for review

**Review by bring-shrubbery — APPROVED** [review]

**Comment by bring-shrubbery** [comment]
Thanks for this PR @dmoroz0v ❤️

**Pull request merged**